README.md

```diff
@@ -1,2 +1,136 @@
 # node-telemetry
 Provides standard Kubernetes probes and Prometheus metrics endpoint for Node.js apps.
+
+## Install
+
+Configure npm with the NES Digital Service GitHub packages repository:
+
+```shell
+echo "@nes-digital-service:registry=https://npm.pkg.github.com" >> .npmrc
+```
+
+Install the module with npm:
+
+```shell
+npm install --save @nes-digital-service/node-telemetry
+```
+
+## Usage
+
+### Kubernetes Lifecycle Probes
+
+In the startup code for your Node.js application add the following code
+
+```js
+const TelemetryServer = require('./node-telemetry')
+
+const TELEMETRY_PORT = 9090
+const telemetry = new TelemetryServer().start(TELEMETRY_PORT)
```

Comment:
**suggestion:** May be worth mentioning that you can pass your logger.

Reply:
Mentioned later in the doc